### DISCUSSION TITLE
DUO Integration Issue: Duo Push Auth failed

### DISCUSSION THREAD

**Discussion by stephenmetzger** [authelia/authelia#3899] [Q&A] [answered]
I think I have everything set up correctly, but when trying to use DUO push, I get the following error in Authelia logs:

```Duo Push Auth failed to process the auth request for <USER> from <PUBLIC_IP>: Invalid signature in request credentials (), error code 40103```

Any thoughts on what might be happening?

**Comment by james-d-elliott** [comment, marked as answer]
This occurs when the secret key is wrong.
